oci/spec_opts_unix.go

```diff
@@ -96,21 +96,21 @@ func WithImageConfig(image Image) SpecOpts {
 		cmd := config.Cmd
 		s.Process.Args = append(config.Entrypoint, cmd...)
 		if config.User != "" {
+			// According to OCI Image Spec v1.0.0, the following are valid for Linux:
+			//   user, uid, user:group, uid:gid, uid:group, user:gid
 			parts := strings.Split(config.User, ":")
```

Comment:
This should use `SplitN` with an argument of 2, right?

Reply:
I'm not sure. I think the current code is ok.